flink/v1.14/flink/src/main/java/com/netease/arctic/flink/read/hybrid/enumerator/ContinuousSplitPlannerImpl.java

```diff
@@ -113,4 +159,20 @@ private ContinuousEnumerationResult discoverInitialSplits() {
         null,
         ArcticEnumeratorOffset.of(changeStartSnapshotId, null));
   }
+
+  protected ContinuousEnumerationResult discoverInitialSplits(List<Expression> filters) {
```

Comment:
These two `discoverInitialSplits` have so much identical code, perhaps we can merge them.